assets/src/edit-story/karma/fixture/fixture.js

```diff
@@ -291,6 +291,14 @@ export class Fixture {
       }
     });
 
+    // Check to see if Roboto font is loaded.
+    await waitFor(async () => {
+      await document.fonts.load('0 Roboto', '');
+      if (!document.fonts.check(' 0 Roboto', '')) {
+        throw new Error('Not ready: Roboto font could not be loaded');
+      }
+    });
```

Comment:
Can we do a similar check for Google Sans on the dashboard fixture?

Reply:
Done https://github.com/google/web-stories-wp/pull/5597/commits/92b9ddcf87dc629b38fbbbae44c1732bdd28d51f